pyfr/plugins/solutionwriter.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+from collections import OrderedDict
+
+from pyfr.h5writer import H5Writer
+from pyfr.inifile import Inifile
+from pyfr.plugins.base import BasePlugin
+
+
+class SolutionWriterPlugin(BasePlugin, H5Writer):
+    name = 'solutionwriter'
+    systems = ['*']
+
+    def __init__(self, intg, cfgsect, *args, **kwargs):
+        BasePlugin.__init__(self, intg, cfgsect)
```

Comment:
There was a problem like this:
```python
class A(object): 
    def __init__(self,a):
        print('A')
        super().__init__(a) # Error on instantiation of e
        # super().__init__() # Error on instantiation of d

class B(object):
    def __init__(self, a):
        print('B')
        super().__init__(a)        

class D(A,B):
    def __init__(self, a):
        print('D')
        super().__init__(5)

class E(A):
    def __init__(self, a):
        print('E')
        super().__init__(5)

d = D(5)
e = E(5)

```
```text
A  B          A
| /       OR  |
D             E
```

I could not locate this situation in rest of PyFR.  
(Diamond inheritance is not an issue though!).